index.html

```diff
@@ -2383,36 +2383,95 @@ <h3>Production</h3>
 in JSON [[RFC8259]] by mapping property values to JSON types as follows:
         </p>
 
-        <ul>
-          <li>
-Numeric values representable as IEEE754 MUST be represented as a
-<a data-cite="RFC8259#section-6">Number type</a>.
-          </li>
-          <li>
-Boolean values MUST be represented as a <a data-cite="RFC8259#section-3">Boolean
-literal</a>.
-          </li>
-          <li>
-Sequence value MUST be represented as an <a data-cite="RFC8259#section-5">Array
-type</a>.
-          </li>
-          <li>
-Unordered sets of values MUST be represented as an <a
-data-cite="RFC8259#section-5">Array type</a>.
-          </li>
-          <li>
-Sets of properties MUST be represented as an <a
-data-cite="RFC8259#section-4">Object type</a>.
-          </li>
-          <li>
-Empty values MUST be represented as a <a data-cite="RFC8259#section-3">null
-literal</a>.
-          </li>
-          <li>
-Other values MUST be represented as a <a data-cite="RFC8259#section-7">String
-type</a>.
-          </li>
-        </ul>
+        <table class="simple" id="json-representation-production">
+          <thead>
+            <tr>
+              <th>
+Data&nbsp;Type
+              </th>
+              <th>
+JSON Representation Type
+              </th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>
+<a data-cite="INFRA#maps">ordered&nbsp;map</a>
+              </td>
+              <td>
+<a data-cite="RFC8259#section-4">JSON Object</a>
```

Comment:
Enumerate the mapping more explicitly:

```suggestion
<a data-cite="RFC8259#section-4">JSON Object</a>, each property is represented as a member of the JSON Object with the property name as the member name and the property value according to its type, as defined in this section
```